### DISCUSSION TITLE
Just fyi It is really confusing that [Jun 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5865] [Discord Threads]
> 🙋‍♂️ **davidpfahler**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Thanks for pointing out we will look into it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1103749829219323996)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
